### PR TITLE
WoE: implement Hire On

### PR DESCRIPTION
**Pull request by strib**
And fix up the chat messages for the other versions of this card.

Issue: #2902

### CODE REVIEW

File: server/game/cards/06-WoE/HireOn.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class HireOn extends Card {
+    // Play: Make a token creature. If there is a combined total of 6
+    // Aember or more between both players' pools, archive Hire On.
+    setupCardAbilities(ability) {
+        this.play({
+            gameAction: [
+                ability.actions.makeTokenCreature(),
+                ability.actions.conditional({
+                    condition: (context) =>
+                        context.player.amber + context.player.opponent.amber >= 6,
```
Comment (romanogit) [inline server/game/cards/06-WoE/HireOn.js:12]: Need to check (context.player.opponent ? context.player.opponent.amber : 0)

File: server/game/cards/06-WoE/HireOn.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    trueGameAction: ability.actions.archive((context) => ({
+                        effect: 'archive {1}',
+                        target: context.source
+                    }))
+                })
+            ],
+            effect: 'make a token creature{1}{2}',
+            effectArgs: (context) =>
+                context.player.amber + context.player.opponent.amber >= 6
```
Comment (romanogit) [inline server/game/cards/06-WoE/HireOn.js:21]: Need to check (context.player.opponent ? context.player.opponent.amber : 0)

File: server/game/cards/06-WoE/HireOn.js (hunk 1 above)
Comment (strib) [reply server/game/cards/06-WoE/HireOn.js:12]: Is that to handle one-player games?

**Comment by strib** [comment]
Updated, thanks for the tip!

File: server/game/cards/06-WoE/HireOn.js (hunk 1 above)
Comment (romanogit) [reply server/game/cards/06-WoE/HireOn.js]: Yes

**Pull request merged**